resources/lib/ui/control.py

```diff
@@ -181,6 +189,7 @@ def xbmc_add_player_item(name, url, iconimage='', description='', draw_cm=None):
     ok=True
     u=addon_url(url)
     cm = draw_cm(addon_url, name) if draw_cm is not None else []
+    cm.append(("List Sources", "RunPlugin(%s)" % addon_url("list_sources/"+u)))
```

Comment:
This should be added from default.py cm callback and not here.
The idea of control.py is to be easily used upon addons.
This line here makes it relay on endpoints which are part of default.py